etc/bash_completion.d/juju2

```diff
@@ -1,42 +1,55 @@
+#!/bin/bash
 # juju-core.bash_completion.sh: dynamic bash completion for juju cmdline,
 # from parsed (and cached) juju status output.
 #
 # Author: JuanJo Ciarlante <jjo@canonical.com>
 # Copyright 2013+, Canonical Ltd.
 # License: GPLv3
 #
+# For juju2, includes --model and --controller handling:
```

Comment:
Per the recent mail to #juju-dev, the binary name will just be 'juju'. I was looking at this script for the packaging, and think what I need is a top level variable JUJU=juju that I can replace when installing, so the wrapping scripts 'juju-1' and 'juju-2.0' can also get correct completetion goodness.